seacrowd/sea_datasets/filipino_hatespeech_tiktok/filipino_hatespeech_tiktok.py

```diff
@@ -0,0 +1,129 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{hernandez2021bert,
+  title={A bert-based hate speech classifier from transcribed online short-form videos},
+  author={Hernandez Urbano Jr, Rommel and Uy Ajero, Jeffrey and Legaspi Angeles, Angelic and Hacar Quintos, Maria Nikki and Regalado Imperial, Joseph Marvin and Llabanes Rodriguez, Ramon},
+  booktitle={2021 5th International Conference on E-Society, E-Education and E-Technology},
+  year={2021}
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["tgl", "eng"]
+_DATASETNAME = "filipino_hatespeech_tiktok"
+
+_DESCRIPTION = """\
+The dataset contains annotated hate speech from transcribed Tiktok videos, mostly in Taglish (codemixed Tagalog and Cebuano) collected via an unofficial Tiktok API.
+Most of the domain and context of the videos are related to politics and general elections in the Philippines. Labeling: 0 - no hate speech, 1 - recognized hate speech
+"""
+
+_HOMEPAGE = "https://github.com/imperialite/filipino-tiktok-hatespeech"
+_LICENSE = Licenses.CC_BY_SA_4_0.value
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_URL = "https://raw.githubusercontent.com/imperialite/filipino-tiktok-hatespeech/main/data"
+
+
+class FilipinoHateSpeechTikTok(datasets.GeneratorBasedBuilder):
```

Comment:
pls add `Dataset` suffix on the class name, for conformity
```suggestion
class FilipinoHateSpeechTikTokDataset(datasets.GeneratorBasedBuilder):
```